src/ServiceProvider.php

```diff
@@ -129,8 +141,28 @@ protected function bootModelEventListeners()
         Runway::allResources()
             ->map(fn ($resource) => get_class($resource->model()))
             ->each(function ($class) {
-                Event::listen('eloquent.saved: '.$class, fn ($model) => Search::updateWithinIndexes(new Searchable($model)));
-                Event::listen('eloquent.deleted: '.$class, fn ($model) => Search::deleteFromIndexes(new Searchable($model)));
+                Event::listen('eloquent.saved: ' . $class, fn ($model) => Search::updateWithinIndexes(new Searchable($model)));
+                Event::listen('eloquent.deleted: ' . $class, fn ($model) => Search::deleteFromIndexes(new Searchable($model)));
             });
     }
+
+    protected function permissionLabel($permission, $resource)
+    {
+        $translationKey = "runway.permissions.{$permission}";
+
+        $label = trans($translationKey, [
+            'resource' => $resource->name(),
+        ]);
+
+        if ($label == $translationKey) {
```

Comment:
I've added an update script & documented the changes in the changelog.

Do we still need this `match` stuff here?

Reply:
I believe we do. If the user doesn't create a `resources/lang/{lang}/runway.php` file, this match is what will ensure that we show the default permissions labels in English. Otherwise, in the CP, we wouldn't even see the permission like name like `create resource`, but we would see `runway.permissions.create`, which is the key for the label in the language file.

Reply:
Gotcha!